common/app/common/GuLogging.scala

```diff
@@ -32,15 +33,15 @@ trait GuLogging {
   // Transparent error logging on exceptions: log context and exception on error, and pass on the exception
   def errorLoggingF[A](context: String)(task: => Future[A])(implicit ec: ExecutionContext): Future[A] = {
     Try(task) match {
-      case Success(f) => f.failed.foreach(Logger.error(context, _)); f
-      case Failure(e) => Logger.error(context, e); throw e
+      case Success(f) => f.failed.foreach(logger.error(context, _)); f
+      case Failure(e) => logger.error(context, e); throw e
```

Comment:
If `Logger`’s already in scope, would it not be more consistent to just use `Logger.logger`?

```suggestion
      case Success(f) => f.failed.foreach(Logger.logger.error(context, _)); f
      case Failure(e) => Logger.logger.error(context, e); throw e
```